sympy/series/sequences.py

```diff
@@ -291,6 +293,78 @@ def __getitem__(self, index):
             return [self.coeff(self._ith_point(i)) for i in
                     range(start, stop, index.step or 1)]
 
+    def find_linear_recurrence(self,n,d=None,gfvar=None):
+        """
+        Finds the shortest linear recurrence that satisfies the first n
+        terms of sequence of order `\leq` n/2 if possible.
+        If d is specified, find shortest linear recurrence of order
+        `\leq` min(d, n/2) if possible.
+        Returns list of coefficients or ``[]`` if no recurrence was found.
```

Comment:
>The coefficients are returned as [b(1), b(2), ...] as defined in the recurrence relation: a(n) = b(1)*a(n-1) + b(2)*a(n-2) + ...

It would be good to include this information in the docstring.